lib/chartmogul/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ChartMogul
-  VERSION = '1.6.6'
+  VERSION = '1.6.7'
```

Comment:
Let's move to [semantic versioning](http://semver.org) It's minor change and version should be
```suggestion
  VERSION = '1.7.0'
```
 WDYT?

Reply:
Fine with 1.7.0 but it was a bug fix originally. Do you still prefer to bump it to 1.7.0?